File: android/language-fallbacks/IgnoreFallback.java
// DocSection: language_fallbacks_ignore
// Tip: Find more about JavaRx SDK at https://docs.kontent.ai/androidandroid

// Prepares the DeliveryOptions configuration object
DeliveryOptions options = DeliveryOptions.builder()
    .projectId("<YOUR_PROJECT_ID>")
    .build();

// Initializes a DeliveryClient for Java projects
DeliveryClient client = new DeliveryClient(options);

// Registers the model class for articles
client.registerType(Article.class);

// Gets the Spanish variant of all content items (while ignoring language fallbacks) using a simple request
CompletionStage<ContentItemsListingResponse> articles = client.getItems(DeliveryParameterBuilder.params()
    .language("es-ES")
    .filterEquals("system.language", "es-ES")
    .build()
);

// Gets the Spanish variant of all content items (while ignoring language fallbacks) using RxJava
Observable.fromCompletionStage(articles).subscribe(new Observer<ContentItemsListingResponse>() {
    @Override
    public void onSubscribe(@NonNull Disposable d) {
    }

    @Override
    public void onNext(@NonNull ContentItemsListingResponse items) {
        // Gets the content items
        List<ContentItem> article = items.getItems();
    }

    @Override
    public void onError(@NonNull Throwable e) {
    }

    @Override
    public void onComplete() {
    }
});
// EndDocSection

File: ios/language-fallbacks/IgnoreFallback.swift
// DocSection: language_fallbacks_ignore
import KenticoCloud
 
let client = DeliveryClient.init(projectId:"975bf280-fd91-488c-994c-2f04416e5ee3")
 
let customQuery = "items?language=es-ES&system.language=es-ES"

// More about strongly typed models https://github.com/Kentico/cloud-sdk-swift#using-strongly-typed-models
client.getItems(modelType: Article.self, customQuery: customQuery) { (isSuccess, itemsResponse, error) in
      if isSuccess {
           if let articles = itemsResponse?.items {
               // Use your items here
           }
       } else {
           if let error = error {
               print(error)
           }
       }
// EndDocSection
